app/javascript/components/Home.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function Home() {
+  return (
+    <main>
+      <h1>Welcome to greetings app</h1>
+      <Link to="/greeting">Click here to view a random Greeting</Link>
```

Comment:
- [ ] The two apps are not properly connecting together and random greetings are not display. You'll required to kindly fix this as you can see on the record below: The `/greeting` link  which lead to localhost/greetings has nothing to do with your RESTfull architecture. We are supposed to randomly display greetings from `http://localhost:3000/api/v1/greetings` We should be making queries and fetch request from the db to display these greetings randomly :+1: 


[Screencast from 08-24-2023 03:44:54 AM.webm](https://github.com/prius2055/hello-rails-react/assets/63087491/3ede29ae-f57e-4048-85bf-eff9fe4c7a56)

Reply:
This has been fixed. Thank you